var/spack/repos/builtin/packages/hto4l/package.py

```diff
@@ -11,7 +11,7 @@ class Hto4l(MakefilePackage):
     up to NLOPS electroweak accuracy and in presence of dimension-6 operators."""
 
     homepage = "https://www2.pv.infn.it/~hepcomplex/hto4l.html"
-    url = "http://www2.pv.infn.it/~hepcomplex/releases/hto4l/Hto4l-v2.02.tar.bz2"
+    url = "https://www2.pv.infn.it/~hepcomplex/releases/hto4l/Hto4l-v2.02.tar.bz2"
```

Comment:
The homepage does not exist.
I did not succeed to `curl -LO` sources but the `spack stage` succeeded. It fetch from `https://mirror.spack.io`

Reply:
It's at https://www2.pv.infn.it/hepcomplex/releases/hto4l/Hto4l-v2.02.tar.bz2 now, no tilde. Changed url, checked hash is unchanged. Although now this server has an invalid ssl certificate chain. You just can't win...